WcaOnRails/app/assets/stylesheets/navbar-static-top.scss

```diff
@@ -56,23 +36,29 @@
     margin-bottom: -($nav-avatar-thumbnail-size - 20px) / 2;
   }
 
+  .search-container{
+    width: 100%;
+    padding: 10px;
```

Comment:
This fixed padding is that causes the bottom gap here:

![image](https://user-images.githubusercontent.com/17034772/100548362-5a6ece00-326c-11eb-8543-d9113ad970c0.png)

A better approach to center the search vertically would be to modify the [`.navbar-right`](https://github.com/thewca/worldcubeassociation.org/blob/bf251da40a157f4e61663929b20aaa2d9b19f64e/WcaOnRails/app/views/layouts/_navigation.html.erb#L153) to display its children centered vertically using:

```css
display: flex;
align-items: center;
```